CHANGELOG.md

```diff
@@ -2,6 +2,103 @@
+    - Added image tag names for Containers in vim performance state.
+    - Added a 'fixed_compute_metric' column to chargeback
+    - Added rate assigning by tags to container images
+    - Chargeback vm group by tag
+  - Tenancy
+    - Prevent deleting mapped tenants from cloud provider
+    - Add checkbox "Tenant Mapping Enabled" to Openstack Manager
+- Providers
+  - Generate a csv of features supported across all models
+  - Containers
+    - Allow policies to prevent Container image scans
+    - Keep pod-container relationship after disconnection
+  - Middleware (Hawkular)
+    - Change labels in middleware topology
+    - Adds "Server State" into Middleware Server Details
+    - Enable search for Middleware entities
+  - Microsoft Cloud (Azure)
+    - Load Balancer inventory collection for Azure
+  - Microsoft Infrastructure (SCVMM)
+    - Set CPU sockets and cores per socket
+  - Network: Nuage enterprises policy groups added
+  - OpenStack
+    - Add hardware state for Hosts
+    - Map Flavors to Cloud Tenants during Openstack Cloud refresh
+  - Red Hat Enterprise Virtualization
+    - Report manufacturer and product info for RHEVM hosts
+  - Storage
+    - New Swift Storage Provider
+    - Enable cinder to handle cloud volume backup
+  - VMware Cloud
+    - Add VMware cloud orchestration stack operations
+  - VMware Infrastructure
+    - Add types and methods for VM Storage Profile
+- REST API
+  - Publish Blueprint API
+  - Update Blueprint API to store bundle info in ui_properties
+  - CRUD for Service create and orchestration template
+  - Api for refreshing automate domain from git
+  - Allow compressed IDs in `service_template_href` and resource { "id" } Api options method
+- SmartState
+  - Containers: Settings for proxy environment variables
+- User Interface
+  - Add GUID validation for certain Azure fields in Cloud Provider screen
+  - OpenStack: Register new Ironic nodes through Mistral
+  - Subnet provisioning for Network provider
+
+
+### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+47+Ending+Oct+3%2C+2016%22+label%3Aenhancement)
+
+- Automate: Changed Automate import to enable system domains.
+- Performance: Do not reload miq server in tree builder
+- REST API: Update API CLI to support the HTTP OPTIONS method.
+- User Interface
+  - Updated PatternFly to v3.11.0
+  - Summary Screen styling updates
+
+### [Fixed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+47+Ending+Oct+3%2C+2016%22+label%3bugs)
+
+Notable fixes include:
+
+- Automate: Fixed problem with request_pending email method
+- Platform
+  - Use correct adjustment in chargeback reports
+  - Replication: Fix typo prevention full error message
+  - Tenancy: User friendly tenant names
+- Providers
+    - Catch unauthorized exception in refresh of openstack
+    - Fix Middleware Operation timeout parameter fetch
+- Provisioning: VMware Infrastructure: sysprep_product_id set to required false
```

Comment:
"Provisioning: VMware Infrastructure: sysprep_product_id field is no longer required"